modules/ossm-rn-new-features.adoc

```diff
@@ -15,6 +15,139 @@ Module included in the following assemblies:
 
 This release adds improvements related to the following components and concepts.
 
+== New features {SMProductName} version 2.5
+
+This release of {SMProductName} adds new features, addresses Common Vulnerabilities and Exposures (CVEs), contains bug fixes, and is supported on {product-title} 4.13 and later versions.
+
+This release ends maintenance support for OpenShift {SMProductShortName} version 2.2. If you are using OpenShift {SMProductShortName} version 2.2, you should upgrate to a supported version.
+
+=== Component versions for {SMProductName} version 2.5
+
+|===
+|Component |Version
+
+|Istio
+|1.18.5
+
+|Envoy Proxy
+|1.26.8
+
+|Kiali
+|1.73.4
+|===
+
+=== Istio 1.18 support
+
+Service Mesh 2.5 is based on Istio 1.18, which brings in new features and product enhancements. While {SMProductName} supports many Istio 1.18 features, the following exceptions should be noted:
+
+* Ambient mesh is not supported
+* QuickAssist Technology (QAT) PrivateKeyProvider in Istio is not supported
+
+=== Cluster-Wide mesh migration
+
+This release adds documentation for migrating from a multitenant mesh to a cluster-wide mesh. For more information, see the following documentation:
+
+* About migrating to a cluster-wide mesh
+* Excluding namespaces from a cluster-wide mesh
+* Defining which namespaces receive sidecar injection in a cluster-wide mesh
+* Excluding individual pods from a cluster-wide mesh
```

Comment:
[Per the ISG](https://www.ibm.com/docs/en/ibm-style?topic=references-citations#ibm-documentation), I think if you can't link to the doc, the title should be placed in quotation marks:
```suggestion
* "About migrating to a cluster-wide mesh"
* "Excluding namespaces from a cluster-wide mesh"
* "Defining which namespaces receive sidecar injection in a cluster-wide mesh"
* "Excluding individual pods from a cluster-wide mesh"
```

Reply:
Done